libs/base/ui/navigation/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -0,0 +1,10 @@
+import { html, LitElement, TemplateResult } from 'lit';
+import { breadcrumbsStyles } from './breadcrumbs.styles';
+
+export class BreadcrumbsComponent extends LitElement {
+  static styles = breadcrumbsStyles;
+
+  protected override render(): TemplateResult {
+    return html`<slot></slot>`;
```

Comment:
You are right. But I tried different approaches, with props and without and all of them have their own drawbacks or limitations. 
For example **dividers**, they are several between breadcrumbs links, and only one thing that we can do to customise them - specify the type of the icon, that has poor flexibility (customer can not provide its own divider or provide different types of them for single breadcrumbs, or if the design does not have dividers between links). Also populating of breadcrumbs like a property has many limitations, for example we are limited to use our own links components, only that hardcoded inside components, or if we need to render icon instead text for the main **Home** breadcrumb.

Slotting of the links the most flexible and simple way. Yes, in result we get just a styled component, but I believe its enough for the main purposes. Developers have enough flexibility to make some customisations, or if need something more complex - use their own solution. The main part - resolving the breadcrumbs date will be the responsibility of another component.   

I will provide a little doc to make component's purpose a bit more clear

Reply:
I understand drawbacks and limitations, but I think in the most of the cases the usage of this components will be really simple - just one divider of the same type or no divider and a few link/text nodes. That's it. And in rare cases when customer will need to customize rendering he can use the slot to replace the whole template or even extend this component.

But as for me it's really important to support an simple usage of our components for easy cases, so I'd put some effort and provide some rendering logic in this component to not force customers provide templates by themselfs, especially keeping in mind that we are still limiting their html with our default css.

In case you're still not convenced - let's have a call, or we can have it together with @dunqan to have a side view.